src/strategies/in-memory-storage.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const {NOT_FOUND} = require('../utils/constants');
+const BaseStorage = require('../base-storage');
+
+export class InMemoryStrategy extends BaseStorage {
+  constructor() {
+    super();
+    this.memoryMap = {};
+  }
+
+  setItem(key, value, options = {}) {
+    if (typeof this.memoryMap[options.namespace] === 'undefined') {
+      this.memoryMap[options.namespace] = {};
+    }
+    this.memoryMap[options.namespace][key] = value;
+    return Promise.resolve();
+  }
+
+  getItem(key, options = {}) {
+    const data = this.memoryMap[options.namespace] && this.memoryMap[options.namespace][key];
+    return data ? Promise.resolve(data) : Promise.reject(NOT_FOUND);
```

Comment:
What if `data === false`? It will be rejected instead of being resolved with 'false' value? Doesn't it? I think, i should be able to set Boolean values. Or everything should be stringified before setting to storage.

Reply:
@shlomitc please also fix this